plugin/clone/src/clone_plugin.cc

```diff
@@ -85,6 +87,20 @@ uint clone_restart_timeout;
 /** Clone system variable: time delay after removing data */
 uint clone_delay_after_data_drop;
 
+/** Clone system variable: list of plugins that will not be matched on
+recipient */
+char *clone_exclude_plugins_list;
+
+/** List of plugins that cannot be excluded by clone_exclude_plugins_list */
+static std::vector<std::string> disallow_list{"daemon_keyring_proxy_plugin",
+                                              "binlog",
+                                              "performance_schema",
+                                              "memory",
+                                              "innodb",
+                                              "keyring_file",
+                                              "keyring_vault",
+                                              "group_replication"};
```

Comment:
I think Simon wants group_replication plugin to NOT be in this disallow_list. 
"
I use native cloning for cloning from non-GR servers (and this works fine) so the requirement of the group_replication plugin to be installed seems to be unnecessary. "

Can we remove the group_replication from this list?